clang/lib/Headers/cheri.h

```diff
@@ -40,8 +40,8 @@ typedef __intcap_t intcap_t;
 typedef __uintcap_t uintcap_t;
 #ifdef WANT_CHERI_QUALIFIER_MACROS
 #define capability __capability
-#define output __output
-#define input __input
+#define output __cheri_output
+#define input __cheri_input
```

Comment:
We should probably kill those defines since they will conflict with lots of variable names.

Reply:
These are guarded behind the (never used?) `WANT_CHERI_QUALIFIER_MACROS`

Reply:
Yeah that's fine. We probably need to update the header anyway